### PR TITLE
Use URL in clipboard, if available, to create a new link

### PR DESCRIPTION
**Pull request by SergioEstevao**
## Description

Related to: https://github.com/wordpress-mobile/gutenberg-mobile/issues/680

This PR changes the link code in React Native to check if the clipboard has an URL and use it if editing a new link on the content.

## How has this been tested?

This can be tested using this GB-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1537

## Types of changes

New feature

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate.

### CODE REVIEW

File: packages/format-library/src/link/index.native.js [hunk 1]
```diff
@@ -108,10 +110,25 @@ export const link = {
 			speak( __( 'Link removed.' ), 'assertive' );
 		}
 
+		getURLFromClipboard = async () => {
```
Comment (hypest) [inline packages/format-library/src/link/index.native.js]: [The build complains](https://travis-ci.com/WordPress/gutenberg/jobs/252800722#L680) that Class Properties is not enabled so, let's change this to:
```suggestion
		async getURLFromClipboard()
```

**Review by hypest — COMMENTED** [review]
Only left one comment/suggestion to fix the build, but otherwise this looks good to me!

**Comment by SergioEstevao** [comment]
@hypest I have updated the code, can you give it a look and approve if all is ok?

File: packages/format-library/src/link/index.native.js [hunk 2]
```diff
@@ -108,10 +111,25 @@ export const link = {
 			speak( __( 'Link removed.' ), 'assertive' );
 		}
 
+		async getURLFromClipboard() {
+			const clipboardText = await Clipboard.getString();
+			if ( ! clipboardText ) {
+				return '';
```
Comment (koke) [inline packages/format-library/src/link/index.native.js]:
```suggestion
				return;
```

File: packages/format-library/src/link/index.native.js [hunk 3]
```diff
@@ -108,10 +111,25 @@ export const link = {
...
+			}
+			// Check if pasted text is URL
+			if ( ! isURL( clipboardText ) ) {
+				return '';
```
Comment (koke) [inline packages/format-library/src/link/index.native.js]:
```suggestion
				return;
```

Nothing is reading a return value from this function

**Review by koke — COMMENTED** [review]
I think the code looks good, I'll test later today

**Review by SergioEstevao — COMMENTED** [review]
Test

**Review by koke — APPROVED** [review]
Tested and it works nicely 👌

**Pull request merged**